### PR TITLE
Use correct API for team members

### PR DESCRIPTION
**Pull request by jjdelc**

### CODE REVIEW

**Comment by jjdelc** [comment]
@mathiasbc I couldn't find the tests associated with the teams helpers to update/fix, do we have any?

**Comment by mathiasbc** [comment]
@jjdelc: No. Since it is now a shared helper for Projects and Teams is worth adding it.

**Comment by jjdelc** [comment]
Tests added https://github.com/Crunch-io/scrunch/pull/292/commits/44898629665e1d3a4886ac1d0840f645632bcb3c

File: scrunch/tests/test_teams.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        self.assertEqual(patch_request.method, 'PATCH')
+        self.assertEqual(json.loads(patch_request.body), {
+            user_url: None
+        })
+
+    def test_list(self):
+        session, team = self.make_team()
+        team_members_url = 'http://example.com/api/teams/ID/members/'
+        team.members.list()
+        patch_request = session.requests[-1]
```
[the remark this answers is not in the document]
Comment (jjdelc) [reply scrunch/tests/test_teams.py]: You're right!

File: scrunch/tests/test_teams.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import json
+from mock import Mock
+from unittest import TestCase
+
+from scrunch.datasets import Team
+from scrunch.datasets import User
+from .mock_session import MockSession
+
+
+class TestTems(TestCase):
```
Comment (dianaclarke) [inline scrunch/tests/test_teams.py]: s/Tems/Teams/

Comment (jjdelc) [reply scrunch/tests/test_teams.py]: Fixed!

**Pull request merged**